config/examples/backend_redis.yml

```diff
@@ -1,9 +1,10 @@
 base: &default
   url: "<%= ENV.fetch('BACKEND_REDIS_URL', 'redis://localhost:6379/6') %>"
   timeout: <%= ENV.fetch('BACKEND_REDIS_TIMEOUT', 1) %>
-  pool_size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
+  size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
   pool_timeout: 5 # this is in seconds
   sentinels: "<%= ENV['BACKEND_REDIS_SENTINEL_HOSTS'] %>"
+  name: "<%= ENV['BACKEND_REDIS_SENTINEL_NAME'] %>"
```

Comment:
I updated the examples to encourage using `:size` over `:pool_size`, and to make customers know about the new `:name` param, which is mandatory when using redis sentinels.

Reply:
Maybe we should set `:name` to some default value (e.g. `default`) if it's missing? 

Do you know if having `""` for name would cause any problems?

I'm looking here https://github.com/redis-rb/redis-client/blob/1ab081c1d0e47df5d55e011c9390c70b2eef6731/lib/redis_client/sentinel_config.rb#L32-L38

and it seems that the error is thrown only if the value is `nil`, but I'm not sure if it can "break" somewhere in another place if the name is empty...

Reply:
We can't set a default because the name is must be a user defined redis sentinel group. The `:name` param is only mandatory when using redis sentinels, it can be `nil` for standalone and for master-replicas redis installations.

Reply:
For clients using redis sentinels, the `:name` param is mandatory now and it will break their configurations if they don't add it. So we must add a release note to mention this.

Reply:
Ah OK, I thought it was only a client thing... 

In that case - wouldn't it be better to keep the name `nil` if `BACKEND_REDIS_SENTINEL_NAME` is missing?

From the link I pasted above, it seems that in this case `name` will be derived from the URL host, and otherwise fail.

But if it is set to empty - will it work as expected?... :thinking: 

This is just theoretical thinking, I haven't tried it myself.

Reply:
I think it makes more sense to set it to `nil` when the env variable isn't set. Although, from my tests in local, I'd say there's no difference in the behavior. Either the name matches exactly the sentinel group name, or it fails, no matter if empty, `nil` or a default value set by `redis-client`.